sql/log_event.cc

```diff
@@ -4962,8 +4962,8 @@ int Query_log_event::do_apply_event(Relay_log_info const *rli,
 #ifdef WITH_WSREP
         rli->report(ERROR_LEVEL, ER_ERROR_ON_MASTER,
                     ER_THD(thd, ER_ERROR_ON_MASTER), expected_error,
-                    (!opt_general_log_raw) && thd->rewritten_query.length()
-                        ? thd->rewritten_query.c_ptr_safe()
+                    (!opt_general_log_raw) && thd->rewritten_query().length()
+                        ? const_cast<String&>(thd->rewritten_query()).c_ptr_safe()
```

Comment:
Please see pxc 5.7 wsrep_mysqld.cc:: wsrep_thd_rewritten_query()
We should use this function returning non-const copy of rewritten query instead of const_casting here (and in several other places)

Reply:
hmm.  that change didn't make it to 8.0 (no wsrep_thd_rewritten_query in 8.0)